xml/admin_gui_dashboard.xml

```diff
@@ -706,28 +721,92 @@
    </para>
   </sect2>
 
+  <sect2 xml:id="dashboard-cluster-inventory">
+   <title>Inventory</title>
+   <para>
+     Click
+     <menuchoice><guimenu>Cluster</guimenu><guimenu>Inventory</guimenu></menuchoice>
+     to view a list of devices. The list includes the device path, type, availability,
+     vendor, model, size, and the OSDs.
+   </para>
+   <para>
+    Click to select a node name in the <guimenu>Hostname</guimenu> column. Once
+    selected, click <guimenu>Identify</guimenu> to identify the device the
+    host is running on. This will initiate an LED light blink. Select the
+    duration of this action between 1, 2, 5, 10, or 15 minutes.
+    Click <guimenu>Execute</guimenu>.
+  </para>
+  <figure>
+   <title>Services</title>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="dash_inventory.png" width="70%" format="PNG"/>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="dash_inventory.png" width="70%" format="PNG"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+ </sect2>
+
   <sect2 xml:id="dashboard-cluster-monitors">
    <title>&mon;s</title>
    <para>
     Click
     <menuchoice><guimenu>Cluster</guimenu><guimenu>Monitors</guimenu></menuchoice>
```

Comment:
```suggestion
    <menuchoice><guimenu>Cluster Monitors</guimenu></menuchoice>
```

Reply:
Just a suggestion...?

Reply:
No, you click "Cluster" first, and then "Monitor" - they are separate sections. Any better way to indicate that, I'm open

Reply:
Nope -- this is the correct way to do it. Leave as is.